common/build.gradle.kts

```diff
@@ -58,6 +58,7 @@ dependencies {
     implementation(libs.androidx.lifecycle.runtime.ktx)
     implementation(libs.androidx.lifecycle.viewmodel.ktx)
     implementation(libs.androidx.media)
+    api(libs.core.remoteviews)
```

Comment:
does this need to be included in the common module? I dont seem to see it used in any common module files

Reply:
You're right, it's only actually needed in :app and :automotive. I've now moved the declaration to each module.